packages/server/lib/util/cookies.ts

```diff
@@ -28,10 +31,23 @@ export const toughCookieToAutomationCookie = (toughCookie: Cookie, defaultDomain
 }
 
 export const automationCookieToToughCookie = (automationCookie: AutomationCookie, defaultDomain: string): Cookie => {
+  let expiry: Date | undefined = undefined
```

Comment:
So its assumed all cookies have no expiration if one isn't defined?

Reply:
that is correct!